### DISCUSSION TITLE
When I attempt to enter sudo mode, it just reloads the page.

### DISCUSSION THREAD

**Discussion by nikoniko78** [community/community#144933] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was trying to give one of my collaborators an assignment to help relocate some files, but this happened. it keeps happening no matter what.

**Comment by Amrit1604** [comment, marked as answer]
When you try to use sudo, the page reloads because GitHub’s web environments (like Codespaces) often block sudo commands for security reasons. It’s their way of preventing users from making system-level changes that could affect the environment.



How to Fix It:
Check Your Permissions: Make sure you have admin rights on the repo. Collaborators without admin access often can’t use sudo in these web terminals.

Use a Local Setup Instead: Clone the repo to your local machine and use your own terminal. This way, you’ll have full access to run sudo without restrictions:


```
git clone https://github.com/your-repo-url.git
cd your-repo
sudo mv /path/to/source /path/to/destination
```
Try Running the Command Directly: Instead of entering sudo mode (sudo -i), run the command in one go:


`sudo mv your-file new-location`
Browser Issue?: If it keeps reloading, it might be a session or browser issue. Clear your cache or try using incognito mode.


If and only if it still doesn't work:
You might need to contact GitHub Support if the issue persists, as it could be a restriction on the environment you’re using.
